test-specifications/token-service/tokenCreateTransaction.md

```diff
@@ -0,0 +1,1150 @@
+
+### Method Name
+
+`createToken`
+
+### Input Parameters
+
+| Parameter Name          | Type                                             | Required/Optional | Description/Notes                                                                                                                           |
+|-------------------------|--------------------------------------------------|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| name                    | string                                           | optional          |                                                                                                                                             |
+| symbol                  | string                                           | optional          |                                                                                                                                             |
+| decimals                | uint32                                           | optional          |                                                                                                                                             |
+| initialSupply           | uint64                                           | optional          |                                                                                                                                             |
+| treasuryAccount         | string                                           | optional          |                                                                                                                                             |
+| adminKey                | string                                           | optional          | DER-encoded hex string representation for private or public keys. Keylists and threshold keys are the hex of the serialized protobuf bytes. |
+| kycKey                  | string                                           | optional          | DER-encoded hex string representation for private or public keys. Keylists and threshold keys are the hex of the serialized protobuf bytes. |
+| freezeKey               | string                                           | optional          | DER-encoded hex string representation for private or public keys. Keylists and threshold keys are the hex of the serialized protobuf bytes. |
+| wipeKey                 | string                                           | optional          | DER-encoded hex string representation for private or public keys. Keylists and threshold keys are the hex of the serialized protobuf bytes. |
+| supplyKey               | string                                           | optional          | DER-encoded hex string representation for private or public keys. Keylists and threshold keys are the hex of the serialized protobuf bytes. |
```

Comment:
There is no need to mention private keys here.  Nobody should _ever_ provide a private key of any type for these fields.
I wonder if we should always require the serialized protobuf type for these keys as a mechanism for encouraging good key practice?

Reply:
The SDKs do allow private keys to be input into transactions. However, the `PrivateKey` objects will never serialize themselves into protobuf fields, it will always serialize its associated `PublicKey`. Sending private keys from the TCK will allow it to test this SDK feature

Reply:
Interesting. Quite unexpected interface choice to allow "sending" a private key (even if it's not actually sent).